crates/turborepo-lib/src/boundaries/mod.rs

```diff
@@ -223,6 +229,18 @@ impl Run {
         })
     }
 
+    fn is_ignored_import(comments: &SingleThreadedComments, span: Span) -> bool {
+        if let Some(import_comments) = comments.get_leading(span.lo()) {
+            for comment in import_comments {
+                if comment.text.trim() == "@boundaries-ignore" {
```

Comment:
What do you think about branding the ignore, and requiring a reason? 

`// @turbo-boundaries-ignore -- we're yolorepo-ing`

A little eslinty, but I think requiring reasons here would be very helpful for maintainers

Reply:
Sure. Do you think we should do `@turbo-boundaries-ignore`? I kinda like just `@boundaries-ignore`